src/codec.rs

```diff
@@ -279,10 +279,10 @@ impl<T: 'static> HasCompact for T where
 
 // compact encoding:
 // 0b00 00 00 00 / 00 00 00 00 / 00 00 00 00 / 00 00 00 00
-//   xx xx xx 00															(0 ... 2**6 - 1)		(u8)
-//   yL yL yL 01 / yH yH yH yL												(2**6 ... 2**14 - 1)	(u8, u16)  low LH high
-//   zL zL zL 10 / zM zM zM zL / zM zM zM zM / zH zH zH zM					(2**14 ... 2**30 - 1)	(u16, u32)  low LMMH high
-//   nn nn nn 11 [ / zz zz zz zz ]{4 + n}									(2**30 ... 2**536 - 1)	(u32, u64, u128, U256, U512, U520) straight LE-encoded
+//   xx xx xx 00															(0 ... 2**6)		(u8)
```

Comment:
then I would write ..= to be more explicit with new syntax

Reply:
oups would be `..` now isn't it

Reply:
oh it doesn't seems to be "up to equal not included" syntax ? in pattern matching

Reply:
`...` is different than `..=`. See https://doc.rust-lang.org/reference/expressions/range-expr.html

Reply:
Yes but .. is different from ..=
but for pattern matching it is a range-pattern: and ... is equals to ..=

https://doc.rust-lang.org/reference/patterns.html#range-patterns

https://play.integer32.com/?version=stable&mode=debug&edition=2018

Reply:
Ahh! Did not thought about that and I also did not know about this change. But the comment is referring to the range and does not use pattern matching. At least in my opinion. Thus, I would not change it.

Reply:
But then I would write `..` to be as rust range no ?

Reply:
Yes. Wtf. My brain is completely fucked up.^^